src/Evolution/Initialization/DiscontinuousGalerkin.hpp

```diff
@@ -30,10 +33,14 @@
 namespace Frame {
 struct Inertial;
 }  // namespace Frame
+namespace Parallel {
+template <typename Metavariables>
+class ConstGlobalCache;
+}  // namespace Parallel
 /// \endcond
 
 namespace Initialization {
-
+namespace Actions {
```

Comment:
I suggest to name this `dg::Actions::InitializeMortars` and place it in `NumericalAlgorithms/DG/Actions`

Reply:
You are welcome to do this in a PR later.

Reply:
sure 👍